### ISSUE TITLE
Add https://ooglester.com

### ISSUE THREAD

**Issue by jeremehancock** [searxng/searx-instances#208]
### Requirements

- [X] It is my instance.
- [X] I give the right to `check.searx.space` to check my instance (every 3 hours for the response times, every 24 hours for the other tests).
- [X] I acknowledge that managing a public instance is not an easy task and require spending time to keep the instance in good health. E.g. look after your instance by using a monitoring system.
- [X] I guarantee to keep an uptime per month of my instance at **minimum 90%**. Please ask for a removal of your instance if there is a planned long downtime or notify us [here](https://github.com/searxng/searx-instances/discussions/categories/report-instance-downtime) for a short downtime.
- [X] I do not track the users of my instance with any analytics or tracking software.
- [X] I won't try to manipulate the ranking of my instance in a way that give an unfair advantage over the other public instances in the list. (e.g. caching requests for searx.space server)

### Bot protection

Yes, I have enabled the limiter plugin built in SearXNG.

**Comment by unixfox** [comment]
Hello @jeremehancock,

This is a friendly reminder that your provider oracle cloud offer IPv6, it would be great if you could add IPv6 to your instance.

It may be not difficult at all, you may already have IPv6 connectivity on your server, all you need is to add a record AAAA in your DNS server.

Here is the documentation here: https://www.51sec.org/2021/09/20/enable-ipv6-on-oracle-cloud-infrastructure/

The future of the internet will thank you, see [why here](https://old.reddit.com/r/explainlikeimfive/comments/1k45y7/eli5_what_is_ipv6_and_why_is_it_so_important/).

**Comment by unixfox** [comment]
Please consider joining our Matrix room for public instance maintainers by joining our Matrix room: https://matrix.to/#/#searxng:matrix.org
then pinging @\unixfox, @\dalf and @\mrpaulblack for asking to be invited to the Matrix room.
We discuss troubles managing a public instance, sharing some advices (like how to protect against bots), announcing big changes in searxng and more.

**Comment by unixfox** [comment]
A new big change has been introduced 🎉!

Please read this documentation for how to proceed: https://gist.github.com/unixfox/4e22730769540fe5b9f1943a86439761

We are going to make mandatory this new parameter by 1st December 2023. Any instance that do not have this parameter enabled won't be allowed in the public list anymore.

We can make exceptions in case you really do not want to enable our bot limiter solution, but you will have to send us proofs: clearly stating how you detect bots, how you actively deal with them and also prove that it really works.

Feel free to ask here if you have any troubles setting up this new parameter.

**Comment by vojkovic** [comment]
Hi @jeremehancock FYI your instance has been down for about a week -> https://github.com/searxng/searx-instances-uptime/issues/13007
